tools/run_tests/xds_k8s_test_driver/framework/test_app/client_app.py

```diff
@@ -386,10 +400,12 @@ def find_server_channel_with_state(
             expected_state=expected_state,
         )
 
-    def get_server_channels(
-        self, server_target: str, **kwargs
+    def find_channels(
```

Comment:
[Minor] This method have only one line, we can probably just use `find_channels_for_target`.

Reply:
Yea, you're right. Let's do it in the next PR tho.